src/cheri_insts.sail

```diff
@@ -868,8 +872,6 @@ union clause ast = StoreCapImm : (regidx, regidx, bits(12))
  *   - *cs1*.**perms** does not grant **Permit_Store**.
  *   - *cs1*.**perms** does not grant **Permit_Store_Capability** and
  *     *cs2*.**tag** is set.
```

Comment:
You should add something here otherwise the comments in the sail code snippet will not be complete

Reply:
But we're not throwing an exception in this case?

Reply:
Aah okay, so the previous comment was wrong

Reply:
Ah, yea, that's the other commit in this PR. :)